packages/ui/src/components/va-viewer/VaViewer.vue

```diff
@@ -0,0 +1,144 @@
+<template>
+  <div
+    class="va-viewer__anchor"
+    v-bind="$attrs"
+  >
+    <slot
+      name="anchor"
+      v-bind="{ openViewer }"
+    />
+    <div
+      v-if="!$slots.anchor"
+      @click="openViewer"
```

Comment:
This must be `.va-viewer-content` (not `.va-viewer__content`, because it is not a child component)